### PR TITLE
0.9.0 release

### PR DESCRIPTION
**Pull request by hasonmsft**
0.9.0 release

### CODE REVIEW

File: ChangeLog.txt [hunk 1]
```diff
@@ -1,6 +1,23 @@
 Note: This is an Azure Storage only package. The all up Azure node sdk still has the old storage bits in there. In a future release, those storage bits will be removed and an npm dependency to this storage node sdk will 
 be taken. This is a CTP v1 release and the changes described below indicate the changes from the Azure node SDK 0.9.8 available here - https://github.com/Azure/azure-sdk-for-node.
 
+2016.03 Version 0.9.0
+
+ALL
+* Updated storage service version to 2015-04-05. Fore more information, please see - https://msdn.microsoft.com/en-us/library/azure/dd894041.aspx
+* Added support for Account SAS.
+* Supported IPACL and Protocols options for service SAS.
```
Comment (yaxia) [inline ChangeLog.txt]: Added support for ...

File: ChangeLog.txt [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+* Fixed the issue where the authentication may fail when a metadata key is in upper case.
+* Added 'nsp check' task for security vulnerability check.
+* Updated the dependency of the 'request' module to avoid the security vulnerability reported by the 'nsp' tool.
+
+BLOB
+* Added new permission 'ADD' for service SAS.
+
+FILE
+* Supported metrics setting for service properties.
```
Comment (yaxia) [inline ChangeLog.txt]: Added support for ...

File: lib/common/util/constants.js [hunk 3]
```diff
@@ -2059,7 +2140,15 @@ var Constants = {
...
 
-  CompatibleVersionConstants: {
+  CompatibleVersionConstants: {    
+    /**
+    * Constant for the 2015-04-05 version.
+    *
+    * @const
+    * @type {string}
+    */
+    APRIL_2015: '2015-04-05',
```
Comment (yaxia) [inline lib/common/util/constants.js]: We don't need it as "2015-04-05" is not a version can be specified.

File: ChangeLog.txt (hunk 2 above)
Comment (hasonmsft) [inline ChangeLog.txt]: Updated and the PR has been re-sent.

File: ChangeLog.txt (hunk 1 above)
Comment (hasonmsft) [inline ChangeLog.txt]: Updated and the PR has been re-sent.

File: lib/common/util/constants.js (hunk 3 above)
Comment (hasonmsft) [inline lib/common/util/constants.js]: Updated and the PR has been re-sent.

**Pull request merged**